src/constants/genesis.ts

```diff
@@ -50,6 +50,18 @@ export const GENESIS: Record<Network, ResponseTypes.Ledger> = {
     max_kes_evolutions: 62,
     security_param: 2160,
   },
+  privnet: {
+    active_slots_coefficient: 0.05,
+    update_quorum: 5,
+    max_lovelace_supply: '45000000000000000',
+    network_magic: 42,
+    epoch_length: 1200,
+    system_start: 1_673_621_531,
+    slots_per_kes_period: 129_600,
+    slot_length: 1,
+    max_kes_evolutions: 62,
```

Comment:
```suggestion
    max_kes_evolutions: 2400,
```

이렇게 올려서 다시 띄운걸로 기억합니다.